.github/workflows/test_e2e.yml

```diff
@@ -15,10 +15,15 @@ on:
   # Allow manual runs
   workflow_dispatch:
 
+env:
+  AWS_REGION: ap-southeast-1
+  MOBILE_E2E_ROLE: ${{ secrets.SALLY_ACTIONS_STG_MOBILE_E2E_ROLE_ARN }}
+  MOBILE_E2E_EXTERNAL_ID: ${{ secrets.SALLY_ACTIONS_STG_MOBILE_E2E_ROLE_EXTERNAL_ID }}
```

Comment:
My bad, after reviewing the permissions I realise that we can re-use the same role for this as well.

so we can do
```
MOBILE_E2E_ROLE: ${{ secrets.SALLY_ACTIONS_STG_E2E_ROLE_ARN }} 
MOBILE_E2E_EXTERNAL_ID: ${{ secrets.SALLY_ACTIONS_STG_E2E_ROLE_EXTERNAL_ID }}
```

Reply:
updated. please review.